.github/workflows/CreatePullRequest.yml

```diff
@@ -8,8 +8,12 @@ jobs:
   assign_and_create_card:
     name: Assign issue to sender and create Kanban card
     runs-on: ubuntu-latest
+    # Single quotes must be used here https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions#literals
     # PRs from forks don't have required token authorization
-    if: github.event.pull_request.head.repo.full_name == github.repository
+    # Dependabot works directly under our repository, but doesn't have enough priviledges to create project card
+    if: |
+        github.event.pull_request.head.repo.full_name == github.repository
+        && github.event.sender.type != 'Bot'
```

Comment:
Bots are recognizable like this, as seen here:
https://github.com/pavel-mikula-sonarsource/GitHubActionPlayground/runs/2829424837?check_suite_focus=true#step:2:461